cldf/cldf-metadata.json

```diff
@@ -17,8 +22,8 @@
         {
             "dc:title": "environment",
             "properties": {
-                "glottolog_version": "v3.3.2",
-                "concepticon_version": "pyconcepticon-1.4.0-10-g1d57531"
+                "glottolog_version": "pyglottolog-1.2.0-3-g0c317cdda7",
```

Comment:
Sorry for barging in - would you mind doing this with Glottolog 4.0?

Reply:
this is -- I've got tag v4.0 checked out, which should be this rev'n?

Reply:
So this is what you get, too:
```bash
glottolog/glottolog$ git describe
v4.0
```
?